sci-astronomy/stellarium/stellarium-0.19.3.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils desktop flag-o-matic xdg virtualx
```

Comment:
Please switch to `cmake.eclass`. With EAPI-7 this should be just a `s/cmake-utils/cmake/` operation.

Reply:
Done.